charts/kafka/charts/zookeeper/templates/statefulset.yaml

```diff
@@ -32,222 +34,198 @@ spec:
       release: {{ .Release.Name }}
       component: server
   updateStrategy:
-{{ toYaml .Values.updateStrategy | indent 4 }}
+    {{- toYaml .Values.updateStrategy | nindent 4 }}
```

Comment:
nindent is the same as indent but prepends a newline to the beginning of the string.
I would argue that indent is correct here.
Does it still make sense to have indent as you have indented the function by hand?

Reply:
I'd prefer if we did away with the indents all together and did them by hand, allows for better readability and editor refactoring...